[2025-08-07 11:36:44,373] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: send_weather_to_kafka.verify_pipeline scheduled__2025-08-07T11:34:37.190902+00:00 [queued]>
[2025-08-07 11:36:44,392] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: send_weather_to_kafka.verify_pipeline scheduled__2025-08-07T11:34:37.190902+00:00 [queued]>
[2025-08-07 11:36:44,392] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-08-07 11:36:44,392] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2025-08-07 11:36:44,392] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-08-07 11:36:44,418] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): verify_pipeline> on 2025-08-07 11:34:37.190902+00:00
[2025-08-07 11:36:44,426] {standard_task_runner.py:52} INFO - Started process 6310 to run task
[2025-08-07 11:36:44,430] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'send_weather_to_kafka', 'verify_pipeline', 'scheduled__2025-08-07T11:34:37.190902+00:00', '--job-id', '209', '--raw', '--subdir', 'DAGS_FOLDER/dag_meteo.py', '--cfg-path', '/tmp/tmpqxe0nq3v', '--error-file', '/tmp/tmp81po88td']
[2025-08-07 11:36:44,432] {standard_task_runner.py:80} INFO - Job 209: Subtask verify_pipeline
[2025-08-07 11:36:44,527] {task_command.py:369} INFO - Running <TaskInstance: send_weather_to_kafka.verify_pipeline scheduled__2025-08-07T11:34:37.190902+00:00 [running]> on host 40571d888a39
[2025-08-07 11:36:44,839] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alla
AIRFLOW_CTX_DAG_ID=send_weather_to_kafka
AIRFLOW_CTX_TASK_ID=verify_pipeline
AIRFLOW_CTX_EXECUTION_DATE=2025-08-07T11:34:37.190902+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-08-07T11:34:37.190902+00:00
[2025-08-07 11:36:44,840] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-08-07 11:36:44,841] {subprocess.py:74} INFO - Running command: ['bash', '-c', '\n            echo " Attente traitement Logstash..." && sleep 30 &&\n            curl -s "http://elasticsearch:9200/_cat/indices/weather*" || echo "Pas encore d\'index weather"\n        ']
[2025-08-07 11:36:44,859] {subprocess.py:85} INFO - Output:
[2025-08-07 11:36:44,862] {subprocess.py:92} INFO -  Attente traitement Logstash...
[2025-08-07 11:37:14,892] {subprocess.py:92} INFO - yellow open weather_data wCNHBp2xSKuPJ9di62P4DQ 1 1 11 0 44.6kb 44.6kb
[2025-08-07 11:37:14,895] {subprocess.py:96} INFO - Command exited with return code 0
[2025-08-07 11:37:14,955] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=send_weather_to_kafka, task_id=verify_pipeline, execution_date=20250807T113437, start_date=20250807T113644, end_date=20250807T113714
[2025-08-07 11:37:14,982] {local_task_job.py:221} WARNING - State of this instance has been externally set to success. Terminating instance.
[2025-08-07 11:37:14,986] {process_utils.py:129} INFO - Sending Signals.SIGTERM to group 6310. PIDs of all processes in the group: [6310]
[2025-08-07 11:37:14,987] {process_utils.py:80} INFO - Sending the signal Signals.SIGTERM to group 6310
[2025-08-07 11:37:15,001] {process_utils.py:75} INFO - Process psutil.Process(pid=6310, status='terminated', exitcode=0, started='11:36:44') (6310) terminated with exit code 0
